modules/api/src/main/java/org/apache/ignite/configuration/schemas/runner/ClusterConfigurationSchema.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.configuration.schemas.runner;
+
+import org.apache.ignite.configuration.annotation.ConfigurationRoot;
+import org.apache.ignite.configuration.annotation.ConfigurationType;
+import org.apache.ignite.configuration.annotation.Value;
+
+/**
+ * Configuration schema for cluster endpoint subtree.
+ */
+@ConfigurationRoot(rootName = "cluster", type = ConfigurationType.DISTRIBUTED)
+public class ClusterConfigurationSchema {
+    /** List of unique names of those cluster nodes that will host distributed metastorage instances. */
+    @Value(hasDefault = true)
+    public String[] metastorageNodes = new String[0];
+
+    /** List of unique names of cluster nodes, which form a baseline. */
+    @Value(hasDefault = true)
+    public String[] baselineNodes = new String[0];
```

Comment:
It also seems to be outdated, currently it's possible to consider all nodes from logic topology as baseline nodes. 

All-in-all, I don't think that we need ClusterConfigurationSchema.